A44-xds-retry.md

```diff
@@ -24,50 +24,50 @@ There are some key differences between gRPC’s and Envoy’s retry policy:
 
 ### Related Proposals
 
-- [A6: gRPC Retry Design](https://github.com/grpc/proposal/blob/master/A6-client-retries.md)
-- [A31: gRPC xDS Timeout Support and Config Selector Design](https://github.com/grpc/proposal/blob/master/A31-xds-timeout-support-and-config-selector.md)
+- [A6: gRPC Retry Design](A6-client-retries.md)
+- [A31: gRPC xDS Timeout Support and Config Selector Design](A31-xds-timeout-support-and-config-selector.md)
 
 ## Proposal
 
 ### Features to support
 
-Due to the differences between gRPC and Envoy, incomplete implementation status of the original [retry gRFC](https://github.com/grpc/proposal/blob/master/A6-client-retries.md) and unresolved [compatibility issues](https://github.com/envoyproxy/envoy/issues/17412) with dynamic cluster selection, we propose initially to support only some of the retry features in Envoy.
+gRPC will support the following xDS retry features:
 
-#### Supported Retry Features
+- Retry on a given set of gRPC status codes ([x-envoy-retry-grpc-on](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/router_filter#x-envoy-retry-grpc-on) values) specified by the `retry_on` config.
+- Backoff retry on retriable error status by a time given by the `retry_back_off` config.
+- Limit the number of attempts that can be retried for a given RPC by the `num_retries` config.
 
-1. Retry on a given set of gRPC status codes (x-envoy-retry-grpc-on values) specified by the retry_on config.
-1. Backoff retry on retriable error status by a time given by the retry_back_off config.
-1. Limit the number of attempts that can be retried for a given RPC by the num_retries config.
+Features that we will not support right now but may add in the future:
 
+- Hedging support, because not all gRPC languages have implemented hedging yet, and we didn't want to wait for that before making retries configurable via xDS.
+- The `per_try_timeout` field, because we can't safely support this now without hedging support, lest we break users when we add hedging support. We will add this later, at the same time as we add hedging support. See the ["Rationale" section](#Supporting-per_try_timeout) below for more details.
+- Any `retry_on` conditions from values of [x-envoy-retry-on](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/router_filter#x-envoy-retry-on).
```

Comment:
I think this line should be deleted.  This is already listed below under "features that we do not plan to support", and I think that's more accurate -- this is not something that we'll be adding later.